### PR TITLE
[WIP] Add golang FVs

### PR DESCRIPTION
**Pull request by lwr20**
Relies on https://github.com/projectcalico/libcalico/pull/154

### CODE REVIEW

File: tests/st/test_pool.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    def test_pool_crud(self):
+        """
+        Test that a basic CRUD flow for pool commands.
+        """
+        with DockerHost('host', dind=False, start_calico=False) as host:
+            # Set up the ipv4 and ipv6 pools to use
+            ipv4_pool = "10.0.1.0/24"
+            ipv6_pool = "fed0:8001::/64"
+
+            ipv4_yaml = """
```
Comment (robbrockbank) [inline tests/st/test_pool.py]: I would probably define these as python dictionaries and write out the dictionaries as json.   The YAML parser handles JSON and YAML so using JSON should suffice for the most part.

File: tests/st/test_pool.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            host.calicoctl("create -f ipv6.yaml", new=True)
+            # And read it back out using the python calicoctl
+            pool_out = host.calicoctl("pool show")
+            # Assert output contains the ipv6 pool, but not the ipv4
+            self.assertNotIn(ipv4_pool, pool_out)
+            self.assertIn(ipv6_pool, pool_out)
+            self.assertNotIn("ipip", pool_out)
+
+            # Now read it out with the Go calicoctl too:
+            pool_out = host.calicoctl("get pool", new=True)
```
Comment (robbrockbank) [inline tests/st/test_pool.py]: I'd recommend using JSON output format and parsing into a dictionary - that way you can do more precise searches than just a simple text search.

File: tests/st/test_pool.py (hunk 1 above)
Comment (lwr20) [inline tests/st/test_pool.py]: We've already got pyyaml as a requirement - makes sense to use dictionaries, but write out using pyyaml.

File: tests/st/test_pool.py (hunk 2 above)
Comment (lwr20) [inline tests/st/test_pool.py]: Or yaml format and read in with pyyaml :)  Agree in principle though.

**Pull request closed without merging**